module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php

```diff
@@ -310,6 +310,30 @@ public function testGetMergedConfigMixinWithFlattening()
         );
     }
 
+    /**
+     * Test getMergedConfig() with string return
```

Comment:
Just curious about name/comment on this test -- not sure what you mean by "with string return" since this is returning an array. Do you mean with string key?